compendium/DeclarativeServices/test/gtest/GlobalServiceTest.cpp

```diff
@@ -0,0 +1,61 @@
+#include "TestFixture.hpp"
+#include "gtest/gtest.h"
+
+#include "../TestUtils.hpp"
+#include "TestInterfaces/Interfaces.hpp"
+
+namespace test
+{
+    struct IntTestService
+    {
+        virtual ~IntTestService() {}
+    };
+
+    struct implTestService : IntTestService
+    {
+    };
+
+    /**
+     * Verify a component with an interface with :: leading can be registered and found
+     */
+    TEST(TestGlobalNamespacing, testServiceWithGlobalNamespaceNoDS)
```

Comment:
is this testpoint needed if `TEST_F(ServiceRegistryTest, TestGlobalNamespaceOnServiceRegistration)` exists? If yes, can this testpoint be put in the core framework test suite instead? I don't see the necessity to have a test that doesn't have to do with DS in the DS test suite.